app/assets/images/payment_icons/zapper.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-zapper">
```

Comment:
Is it possible to add more space around the icon? It looks a bit tight.

[![alt](https://screenshot.click/27-45-j5ax8-w7vzk.png)](https://screenshot.click/27-45-j5ax8-w7vzk.png)

Reply:
Also, can you run your SVG through SVGO? The instructions are in the [Contribution guideline](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md)

Reply:
used our shorter logo, added spacing, ran svgo